### PR TITLE
DOCSP-26387: added a mongocli  backup list example

### PR DESCRIPTION
**Pull request by kyuan-mongodb**
## Proposed changes

_Jira ticket:_ DOCSP-26387

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -17,36 +17,183 @@ cloud.google.com/go v0.65.0/go.mod h1:O5N8zS7uWy9vkA9vayVHs65eM1ubvY4h553ofrNHOb
...
+cloud.google.com/go/aiplatform v1.22.0/go.mod h1:ig5Nct50bZlzV6NvKaTwmplLLddFx0YReh9WfTO5jKw=
+cloud.google.com/go/aiplatform v1.24.0/go.mod h1:67UUvRBKG6GTayHKV8DBv2RtR1t93YRu5B1P3x99mYY=
+cloud.google.com/go/analytics v0.11.0/go.mod h1:DjEWCu41bVbYcKyvlws9Er60YE4a//bK6mnhWvQeFNI=
+cloud.google.com/go/analytics v0.12.0/go.mod h1:gkfj9h6XRf9+TS4bmuhPEShsh3hH8PAZzm/41OOhQd4=
+cloud.google.com/go/area120 v0.5.0/go.mod h1:DE/n4mp+iqVyvxHN41Vf1CR602GiHQjFPusMFW6bGR4=
+cloud.google.com/go/area120 v0.6.0/go.mod h1:39yFJqWVgm0UZqWTOdqkLhjoC7uFfgXRC8g/ZegeAh0=
+cloud.google.com/go/artifactregistry v1.6.0/go.mod h1:IYt0oBPSAGYj/kprzsBjZ/4LnG/zOcHyFHjWPCi6SAQ=
+cloud.google.com/go/artifactregistry v1.7.0/go.mod h1:mqTOFOnGZx8EtSqK/ZWcsm/4U8B77rbcLP6ruDU2Ixk=
+cloud.google.com/go/asset v1.5.0/go.mod h1:5mfs8UvcM5wHhqtSv8J1CtxxaQq3AdBxxQi2jGW/K4o=
+cloud.google.com/go/asset v1.7.0/go.mod h1:YbENsRK4+xTiL+Ofoj5Ckf+O17kJtgp3Y3nn4uzZz5s=
```
Comment (andreaangiolillo) [inline go.sum:46]: you should not change this file

File: internal/cli/atlas/backup/snapshots/list.go [hunk 2]
```diff
@@ -15,6 +15,7 @@
 package snapshots
 
 import (
+	"fmt"
```
Comment (andreaangiolillo) [inline internal/cli/atlas/backup/snapshots/list.go:18]: the file is not formatted. Lint is failing. Run `make lint` to validate against our linting rules.

File: internal/cli/atlas/backup/snapshots/list.go [hunk 3]
```diff
@@ -67,6 +68,8 @@ func ListBuilder() *cobra.Command {
 		Annotations: map[string]string{
 			"clusterNameDesc": "Name of the Atlas cluster that contains the snapshots you want to retrieve.",
 		},
+		Example: fmt.Sprintf(`  # Return a JSON-formatted list of snapshots for the cluster named myDemo 
+		  %s atlas backups snapshots list myDemo --output json`, cli.ExampleAtlasEntryPoint()),
```
Comment (andreaangiolillo) [inline internal/cli/atlas/backup/snapshots/list.go:72]:
```suggestion
		  %s backups snapshots list myDemo --output json`, cli.ExampleAtlasEntryPoint()),
```

**Review by andreaangiolillo — APPROVED** [review]
LGTM

File: internal/cli/atlas/backup/snapshots/list.go [hunk 4]
```diff
@@ -67,6 +68,8 @@ func ListBuilder() *cobra.Command {
 		Annotations: map[string]string{
 			"clusterNameDesc": "Name of the Atlas cluster that contains the snapshots you want to retrieve.",
 		},
+		Example: fmt.Sprintf(`  # Return a JSON-formatted list of snapshots for the cluster named myDemo 
+		  %s backups snapshots list myDemo --output json`, cli.ExampleAtlasEntryPoint()),
```
Comment (sarahsimpers) [inline internal/cli/atlas/backup/snapshots/list.go:72]: Adjust spacing- should be two spaces indented (not from "Example" but from the start of the line)
```suggestion
  %s backups snapshots list myDemo --output json`, cli.ExampleAtlasEntryPoint()),
```

**Review by sarahsimpers — CHANGES_REQUESTED** [review]

**Review by sarahsimpers — APPROVED** [review]
LGTM!

**Pull request merged**